src/components/home/index.jsx

```diff
@@ -11,28 +12,39 @@ import './home.scss';
 
 class Home extends React.Component {
   state = {
-    searchTerm: ""
+    searchTerm: "",
+    gender: ""
   }
   
   handleSubmit = (searchTerm) => {
     this.setState({ searchTerm });
   };
 
+  handleChangeGender = gender => {
+    this.setState({ gender });
+  }
+
   delayedHandleSubmit = _.debounce(this.handleSubmit, 300);
 
   render() {
-    const { searchTerm } = this.state;
+    const { searchTerm, gender } = this.state;
+
     const DashboardQueryRenderer = (
       <QueryRenderer
         environment={environment}
         query={graphql`
-          query homeQuery ($searchTerm: String) {
-            allJobs(jobTitle_Icontains: $searchTerm, before: "12/3/2020") {
+
+          query homeQuery ($searchTerm: String, $gender: String) {
```

Comment:
The name tag for this query should be `HomeQuery`. Relay expects the tag name to begin with the module name and end with the action(Mutation, Query, Subscription).